src/mm-progress-bar/mm-progress-bar.scss

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+@import "_bourbon";
+@import "_color";
+@import "_mixins";
+
+:host {
+	display: block;
+	font-size: inherit;
+	height: auto;
+	width: 100%;
+}
+
+.bar {
+	border-style: solid;
+	border-width: 1px;
+	border-radius: 9999px;
+	box-sizing: border-box;
+	display: block;
+	height: 1em;
```

Comment:
re: height - we've gone with px in just about every other component, so this may be a break with consistency. I'm thinking it should probably be 10px "out-of-the-box"...